res/themes/legacy-dark/css/_legacy-dark.scss

```diff
@@ -14,217 +14,132 @@ $text-secondary-color: #a1b2d1;
 $room-highlight-color: #343a46;
 
 // typical text (dark-on-white in light skin)
-$primary-fg-color: $text-primary-color;
-$secondary-fg-color: $primary-fg-color;
-$tertiary-fg-color: $primary-fg-color;
-$primary-bg-color: $bg-color;
-$muted-fg-color: $header-panel-text-primary-color;
-
-// used for dialog box text
-$light-fg-color: $header-panel-text-secondary-color;
+$primary-fg-color: #edf3ff;
+$primary-bg-color: #181b21;
+$muted-fg-color: #a1b2d1; // Commonly used in headings and relevant alt text
```

Comment:
The hex values are used here since the hex value for these variables are used in `_legacy-light.scss`. (https://github.com/matrix-org/matrix-react-sdk/pull/7726/files/21b8d90ca9cc4eb0e2b9332ccf70575d88fdcb19#diff-90861a924b357562176a919e52e7efbe2f21df686d105dd439fcf244c9d51108R16)

`_legacy-default` contains hardcoded lower level variables (i.e. variables that other variables will be assigned to), `_legacy-light/dark` contain hardcoded lower level variables specified by that specific theme. Higher level variables are assigned to lower level variables via `_legacy-common`. `_legacy-dark` has a specific value of `primary-fg-color` and `_legacy-light` has a specific value of `primary-fg-color` that is unique to them. If a variable needs to be assigned to another variable than that same behavior should be replicated in the other theme, hence moving variable assignments into `_legacy-common`.